### DISCUSSION TITLE
Duplicate Socket Connection with NextJs

### DISCUSSION THREAD

**Discussion by hnhai** [socketio/socket.io#4479] [Q&A]
I'm using NextJs combine with socket.io connect to Socket Server.
When visit website by browser, I see 2 connection connect to socket server (Server Side Tool), but when close browser only one connection close, one another one keep connect.
This issue also apear when I reload browser, connection increment each time reload.
When I shutdown NextJs application. All connection close.
I think both client (browser) and NextJs server connection to Socket Server.
But don't know how to fix this problem. Thanks

**Comment by abhinavdhangar** [comment]
same problem here bro

**Reply by hnhai** [reply]
@abhinavdhangar You can using `dynamic ssr=false` with socket component to avoid nextJs Server connect to socket.
